frontend/src/components/GenericModal.vue

```diff
@@ -35,6 +41,9 @@ onUnmounted(() => {
 <template>
   <div class="new-design overlay" role="dialog" tabindex="-1" aria-labelledby="title" aria-modal="true">
     <div class="modal">
+        <div class="modal-close" v-if="closable" @click="emits('close')" role="button" aria-labelledby="modal-close-button">
+          <img id="modal-close-button" src="@/assets/svg/icons/close.svg" :alt="t('label.close')" :title="t('label.close')"/>
+        </div>
```

Comment:
I guess I could make this a button as well 🤔

Reply:
I agree. Maybe we can use our LinkButton.vue? Or create another component for icon buttons.

Reply:
Oh right I forgot about that already 😅